application/controllers/survey_response.py

```diff
@@ -39,11 +47,15 @@ def add_survey_response(self, case_id, file, file_name, survey_ref):
 
         if self.check_if_file_size_too_small(file_size):
             log.info('File size is too small')
-            return 'False'
-        elif not self.check_if_file_size_too_small(file_size):
-            log.info('File size is correct')
+            raise FileTooSmallError()
+        else:
             json_message = self._create_json_message_for_file(file_name, file_contents, case_id, survey_ref)
-            return send_message_to_rabbitmq_queue(json_message, tx_id, RABBIT_QUEUE_NAME)
+            sent = send_message_to_rabbitmq_queue(json_message, tx_id, RABBIT_QUEUE_NAME)
+            if sent:
+                return sent
+            else:
+                raise SurveyResponseError
```

Comment:
Couple of minor things

1. no need to return sent, as you're not using the value in the calling method anymore
1. missing () on raise SurveyResponseError 
1. might be worth adding a log statement if it fails

Something like
```
 sent = send_message_to_rabbitmq_queue(json_message, tx_id, RABBIT_QUEUE_NAME)
 if not sent:
     log.error("Unable to send file to rabbit queue")
     raise SurveyResponseError()
```